src/chrome/content/rules/yWorks.xml

```diff
@@ -0,0 +1,11 @@
+<ruleset name="yWorks">
+	<target host="www.yworks.com" />
+	<target host="yworks.com" />
+	<target host="www.yworks.de" />
+	<target host="yworks.de" />
+
+	<securecookie host=".+" name=".+" />
+
+	<rule from="^http://(www\.)?yworks\.de/" to="https://www.yworks.com/" />
```

Comment:
Please document the issue.

Reply:
The rule is necessary, both `yworks.de` and `www.yworks.de` fail due to a TLS handshake error.